stdlib/ballerina-builtin/src/main/ballerina/runtime/Package.md

```diff
@@ -0,0 +1,100 @@
+## Package overview
+
+This package includes functions to interact with the runtime, the invocation context and to manage errors.
+
+### Invocation Context
+
+The Invocation Context is a data holder which is created per request, and preserved for a single request-response flow. 
+The Invocation Context comprises of a unique id, UserPrincipal instance which includes user details and an AuthContext 
+instance which has the authentication related details if available.
+
+### Errors
+
+Error types NullReferenceException and IllegalStateException, which wraps the error type 
+defined in ballerina/builtin package, are included in the runtime package. Furthermore, there are utility methods to 
```

Comment:
**Suggestion**
The runtime package includes the 'NullReferenceException' and 'IllegalStateException' error types that wrap the error types defined in the 'ballerina/builtin' package.